### PR TITLE
Change postion.get from sync to async  

### PR DESCRIPTION
**Pull request by pietgeursen**

### CODE REVIEW

File: position/async/position.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+      type: 'position',
+      root: typeof poll === 'string' ? poll : poll.key,
+      details
+    }
+
+    if (reason) content.reason = reason
+
+    if (channel) {
+      if (typeof channel !== 'string') { throw new Error('channel must be a string') }
+      content.channel = channel
```
Comment (mixmix) [inline position/async/position.js:22]: prune # off front?

File: position/async/position.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    if (content.root && server) {
+      pull(
+        pullAsync(cb => {
+          getPoll(content.root, cb)
+        }),
+        pull.drain(({positions}) => {
+          content.branch = sort.heads(positions)
+          cb(null, content)
+        })
+      )
```
Comment (mixmix) [inline position/async/position.js]: this could be a straight cb?

File: position/async/position.js (hunk 1 above)
Comment (pietgeursen) [reply position/async/position.js:22]: done

Comment (pietgeursen) [reply position/async/position.js:22]: woops, not done. wrong comment.

Comment (pietgeursen) [reply position/async/position.js:22]: added this as a different ticket

**Pull request merged**